src/Oci8/Query/Grammars/OracleGrammar.php

```diff
@@ -432,10 +433,12 @@ protected function dateBasedWhere($type, Builder $query, $where)
      */
     protected function wrapValue($value)
     {
-        if ($this->isReserved($value)) {
-            return parent::wrapValue($value);
+        if ($value === '*') {
+            return $value;
         }
 
-        return $value !== '*' ? sprintf($this->wrapper, $value) : $value;
+        $value = $this->isReserved($value) ? Str::lower($value) : Str::upper($value);
```

Comment:
I'm wondering why are reserved words lowercased? It would still wrap the reserved word in quotes, like `"order"`.

For example, if you have a `DB::table('table_name')->select('order')->toSql();` it produces `select "order" from table_name`. Wouldn't you want "order" to be capitalized?